app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -9,7 +9,7 @@ def create_in_provider(manager_id, params)
 
       refresh(manager)
       find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
-    rescue AnsibleTowerClient::ClientError, ActiveRecord::RecordNotFound => error
+    rescue Exception => error
```

Comment:
There's one time a weird constant not recognized error. but I couldn't reproduce it again.